mobile/apps/auction/routes.coffee

```diff
@@ -31,8 +31,7 @@ module.exports.index = (req, res, next) ->
   auction = new Auction id: id
   saleArtworks = new SaleArtworks [], id: id
   artworks = new Artworks
-  artworks.comparator = (artwork) ->
-    (saleArtwork = artwork.related().saleArtwork).get('lot_label') or saleArtwork.id
+  artworks.comparator = (artwork) -> (saleArtwork = artwork.related().saleArtwork).get('position') or saleArtwork.id
```

Comment:
I'll try to come back around for this- at the time we were just trying to fix the display issue.